symfit/core/models.py

```diff
@@ -60,11 +60,20 @@ def __init__(self, variables, output):
         :param output: The output of a call which should be mapped to
             ``variables``.
         """
-        self.variables = variables
+        self.variables = list(variables)
         self.output = output
         self.output_dict = OrderedDict(zip(variables, output))
         self.variable_names = {var.name: var for var in variables}
 
+    def __getnewargs_ex__(self):
```

Comment:
Is it possible to define __getnewargs__ instead and return args instead of kwargs? That should fix the failing tests for py3.5?